platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/events/BaseEventHashedData.java

```diff
@@ -393,4 +483,13 @@ public long getGeneration() {
     public static long calculateGeneration(final long selfParentGeneration, final long otherParentGeneration) {
         return 1 + Math.max(selfParentGeneration, otherParentGeneration);
     }
+
+    /**
+     * Get an event descriptor for this event.
+     *
+     * @return an event descriptor for this event
+     */
+    public EventDescriptor getEventDescriptor() {
```

Comment:
probably should be called `createEventDescriptor` or something, since its not a getter

Reply:
changed